backend/app/models/comment.py

```diff
@@ -0,0 +1,42 @@
+"""A model representing user comments on content."""
+
+from datetime import datetime
+from uuid import uuid4
+from marshmallow_sqlalchemy import SQLAlchemyAutoSchema
+from ..extensions import DB as db
+
+
+# pylint: disable=too-few-public-methods
+class Comment(db.Model):
+    """Comment model representing user comments on content."""
+
+    __tablename__ = "comment"
```

Comment:
use plural form of a word for table names. 

- [ ] use "comments" instead of "comment" as a table name.